config/environments/test.rb

```diff
@@ -13,7 +13,11 @@
   # Do not eager load code on boot. This avoids loading your whole application
   # just for the purpose of running a single test. If you are using a tool that
   # preloads Rails for running tests, you may have to set it to true.
-  config.eager_load = false
+  if ENV['CODECOV_TOKEN']
+    config.eager_load = true
+  else
+    config.eager_load = false
+  end
```

Comment:
I had tried this in the past as well, but it doesn't seem to help :(

Reply:
If you add a `puts` statement like the SimpleCov docs say in their Readme under troubleshooting (https://github.com/colszowka/simplecov#troubleshooting), the files are loaded _after_ simplecov is loaded, which is correct. :shrug: :sob: